### ISSUE TITLE
The battery drains way faster on treble than it was on the stock firmware with low battery power mode enabled

### ISSUE THREAD

**Issue by secretwolf98** [phhusson/treble_experimentations#957]
The battery drains quite too fast on Treble, even with the low battery power saving mode enabled, compared to the stock firmware. My device is the Nook Tablet 10.1" 2018 BNTV650

**Comment by phhusson** [comment]
What do you call high drain?

**Comment by secretwolf98** [comment]
@phhusson down 12% in about 2-5 minutes.

**Comment by phhusson** [comment]
So, after 40 minutes battery is dead?

**Comment by secretwolf98** [comment]
@phhusson, possibly yes.

**Comment by alexa-v2** [comment]
Highly dependant on apps installed, usage, etc

Have you checked logcat? How about battery usage, see what's actually using it? Also keep in mind that your stock firmware is iirc highly customized and optimized

**Comment by secretwolf98** [comment]
@alexa-v2, if stock is so "optimized". Then why is Phh Treble so much faster with performance compared to stock?

**Comment by alexa-v2** [comment]
Because less bloatware and some OEMs throttle the hardware for battery sake
Remember, more performance equals less battery life and give versa

**Comment by phhusson** [comment]
40 minutes battery life sounds like a hardware issue. No matter how buggy the software is.

**Comment by alexa-v2** [comment]
@phhusson taking a wild guess in the dark but if I had to guess, stock has some sort of battery saving/performance throttling crapware on stock, resulting in worse performance but better battery. Seen cheaper LG and Alcatel devices employ similar tactics

Mind you I don't own the device so this is merely a hypothesis and could be completely wrong

**Comment by alexa-v2** [comment]
@secretwolf98 tried going in battery saver all the time? Checked 3rd party apps?

**Comment by phhusson** [comment]
40 minutes of battery life isn't some performance optimization or whatever,
it's really extremely catastrophic
(And i've worked for an OEM doing really shitty devices)

**Comment by secretwolf98** [comment]
@alexa-v2, yep. I have battery saver enabled 24/7 (when the device isn't plugged in)
